sql/executor.go

```diff
@@ -99,6 +99,11 @@ func (e *Executor) Execute(args driver.Request) (driver.Response, int, error) {
 		}
 		planMaker.setTxn(txn, planMaker.session.Txn.Timestamp.GoTime())
 	}
+	var err error
+	planMaker.evalCtx.Location, err = getTimeZone(planMaker.session)
```

Comment:
Any reason my comment to add a `GetLocation` function pointer to `EvalContext` keeps getting ignored? This would look like:

```
type EvalContext struct {
  GetLocation func() (*time.Location, error)
}
```

Then you would initialize as:

```
  planMaker.evalCtx.GetLocation = planMaker.session.getLocation
```